docs/getting_started.md

```diff
@@ -203,6 +203,167 @@ See these resources for more information on clustering:
+
+```python
+# submitter-provided annotations for each cell
+processed_adata.obs["submitter_celltype_annotation"]
+```
+
+Cells that submitters did not annotate are labeled with `NA`.
+Note that submitter-provided annotations are also present in unfiltered and filtered objects and can be accessed using the same approach shown here for processed objects.
+
+
+#### `SingleR` annotations
+
+To access automated `SingleR` annotations in the `SingleCellExperiment`, use the following command:
+
+```r
+# SingleR annotations for each cell
+processed_sce$singler_celltype_annotation
+```
+
+To access automated `SingleR` annotations in the `AnnData` object, use the following command:
+
+```python
+# SingleR annotations for each cell
+processed_adata.obs["singler_celltype_annotation"]
+```
+
+Cells that `SingleR` could not confidently annotate are labeled with `NA`.
+
+
+If `SingleR` annotation was performed using ontology reference labels (default), the cell type annotation ontology terms will also be provided.
+
+To access cell type annotation ontology terms in the `SingleCellExperiment` object, use the following command:
+```r
+# SingleR ontology annotations
+processed_sce$singler_celltype_ontology
+```
+
+To access cell type annotation ontology terms in the `AnnData` object, use the following command:
+```r
+# SingleR ontology annotations
+processed_adata.obs["singler_celltype_ontology"]
+```
+
+
+You can also access the full object returned by `SingleR` from the `SingleCellExperiment`'s metadata with the following command (note that this information is not provided in the `AnnData` object):
+
+```r
+# SingleR full result
+metadata(processed_sce)$singler_results
+```
+
+`SingleR` annotation uses a reference dataset from the [`celldex` package](https://bioconductor.org/packages/release/data/experiment/html/celldex.html) [[Aran _et al._ (2019)](https://doi.org/10.1038/s41590-018-0276-y)].
+
+To access information about the `SingleR` reference in the `SingleCellExperiment` object, use the following commands:
+
+
+```r
+# SingleR reference dataset name
+metadata(processed_sce)$singler_reference
+
+# SingleR reference dataset label used for annotation
+metadata(processed_sce)$singler_reference_label
+
+# SingleR reference version
+metadata(processed_sce)$singler_reference_version
+```
+
+To access the analogous information in the `AnnData` object, use the following commands:
+
+
+```python
+# SingleR reference (celldex) dataset name
+processed_adata.uns["singler_reference"]
+
+# SingleR reference (celldex) dataset label used for annotation
+processed_adata.uns["singler_reference_label"]
+
+# SingleR reference (celldex) version
+processed_adata.uns["singler_reference_version"]
+```
+
+<!-- TODO: this instead? -->
+Additional information about `SingleR` annotation results is also available from the `SingleCellExperiment`'s metadata, as described in the {ref}`experiment metadata table<sce_file_contents:SingleCellExperiment experiment metadata>`.
```

Comment:
Personally, I think this is a little long and we don't give this much detail about metadata in the other sections. I think you replace the version information with a link like you suggest here.

Reply:
I think I'll also be a little more explicit like `Additional information, including reference and version metadata, ...`